test/test-debuglet.js

```diff
@@ -112,6 +113,7 @@ describe('Debuglet', function() {
 
       debuglet.once('registered', function(id) {
         assert.equal(id, DEBUGGEE_ID);
+        assert.equal(debuglet.debuggee_.project, projectId);
```

Comment:
Agreed. Currently the vast majority of comparisons are still `assert.equal`, and I added similar comparisons for uniformity; I think it would be best to change it all at once (in another PR). I don't think it's harmful as of right now since the RHS is almost always a string constant.